database/migrations/2019_11_30_092123_create_post_statuses_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePostStatusesTable extends Migration {
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up() {
+		Schema::create( 'post_statuses', function ( Blueprint $table ) {
```

Comment:
Please follow PSR standards during implementation of laravel applications. No extra spaces within parentheses `()` of methods, braces of methods should be on separate lines.
https://svyatoslav.biz/misc/psr_translation/#_PSR-1
Please configure inspection rules and auto-formatting of your IDE to meet PSR requirements